app/nagbot.py

```diff
@@ -44,9 +44,7 @@ def notify_internal(channel, dryrun):
             summary_msg += f"\n*{resource_type.__name__}s:*\nWe have {num_active_resources} " \
                            f"{ec2_state} {ec2_type}s right now and {num_total_resources} total.\n" \
                            f"If we continue to run these {ec2_type}s all month, it would cost {running_monthly_cost}.\n"
-
-            resources = sorted((r for r in resources if not (len(r.eks_nodegroup_name) > 0)), key=lambda i: i.name)
```

Comment:
This also sorted the resources by name - do we want to keep that part?

Reply:
In prod `notify` is sorted but `execute` is not. So, I took a little liberty to remove it and make it consistent for both.